catalog/transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/xml/adapter/ObjectAdapter.java

```diff
@@ -51,6 +57,14 @@ public static ObjectElement marshalFrom(Attribute attribute) throws CatalogTrans
   public static Attribute unmarshalFrom(ObjectElement element) {
     AttributeImpl attribute = null;
     for (Serializable value : element.getValue()) {
+      try {
+        value =
+            (Serializable)
+                new ObjectInputStream(new ByteArrayInputStream((byte[]) value)).readObject();
```

Comment:
Shouldn't we close the ObjectInputStream?

Reply:
I'll change it to a try with resources